gcc/rust/resolve/rust-early-name-resolver.cc

```diff
@@ -466,7 +466,7 @@ EarlyNameResolver::visit (AST::MacroInvocation &invoc)
   NodeId resolved_node = UNKNOWN_NODEID;
   NodeId source_node = UNKNOWN_NODEID;
   if (has_semicolon)
-    source_node = invoc.get_macro_node_id ();
+    source_node = invoc.get_node_id ();
   else
     source_node = invoc.get_node_id ();
```

Comment:
see for example here we're no longer doing anything different if the macro has a semicolon or not, so we can remove the `if` block